### PR TITLE
Mio 0.6

### LINKED ISSUE #28: Mio 0.6 support

### PR DESCRIPTION
**Pull request by bonifaido**
fixes #28.

### CODE REVIEW

File: examples/zookeeper_example.rs [hunk 1]
```diff
@@ -91,39 +91,40 @@ fn zk_example() {
...
+    // let zk_arc_captured = zk_arc.clone();
+    // thread::spawn(move || {
+    //     zk_arc_captured.close().unwrap();
+
+    //     // And operations return error after closed
+    //     match zk_arc_captured.exists("/test", false) {
+    //         Err(err) => println!("Usage after closed should end up with error: {:?}", err),
+    //         Ok(_) => panic!("Shouldn't happen"),
+    //     }
+    // });
```
Comment (rgs1) [inline examples/zookeeper_example.rs]: ditto

File: examples/zookeeper_example.rs [hunk 2]
```diff
@@ -91,39 +91,40 @@ fn zk_example() {
...
+    //     }
+    // }
+
+    // let (ev_tx, ev_rx) = mpsc::channel();
+    // pcc.add_listener(move |e| ev_tx.send(e).unwrap());
+    // thread::spawn(move || {
+    //     for ev in ev_rx {
+    //         println!("received event {:?}", ev);
+    //     }
+    // });
```
Comment (rgs1) [inline examples/zookeeper_example.rs]: somehow my comment got lost: should we just delete this for now? (remains in git)

Comment (bonifaido) [inline examples/zookeeper_example.rs]: I should rebase this stuff.

**Comment by bonifaido** [comment]
Closed it in favour of #32

**Pull request closed without merging**